src/SIM/AdaptiveSIM.C

```diff
@@ -309,7 +336,11 @@ bool AdaptiveSIM::adaptMesh (int iStep)
   {
     IFEM::cout <<"\nRefining by increasing solution space by "<< beta
                <<" percent."<< std::endl;
-    prm.errors = eNorm.getRow(eRow);
+    std::vector<DblIdx> errors(model.getPatch(1)->getNoNodes(1));
+    remapErrors(errors, model.getPatch(1), eNorm.getRow(eRow));
```

Comment:
But is this necessary in the non-mixed case then? How about the `remapErrors` method is virtual in ASMbase and with the above implementations in ASMu2Dmx and ASMu3Dmx, respectively, and one-to-one mapping in ASMbase?